controller/volume_controller.go

```diff
@@ -1553,6 +1639,269 @@ func getRebuildingReplicaCount(e *longhorn.Engine) int {
 	return rebuilding
 }
 
+type replicaAutoBalanceCount func(*longhorn.Volume, map[string]*longhorn.Replica) (int, map[string][]string, error)
+
+func (vc *VolumeController) replicaAutoBalanceLeastEffort(v *longhorn.Volume,
```

Comment:
Suggest changing to `getReplicaCountForAutoBalanceLeastEffort`

Reply:
Addressed.